src/server/game/DungeonFinding/LFGQueue.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2008-2015 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2008-2014 TrinityCore <http://www.trinitycore.org/>

```

Comment:
I know. I just was so tired when I finished this, after browsing through 1787 commits, that I fell asleep directly and did not wake up until now, couldn't think about it until now. :stuck_out_tongue_closed_eyes: